contrib/valkey-go/valkey.go

```diff
@@ -0,0 +1,369 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016 Datadog, Inc.
+
+// Package valkey provides tracing functions for tracing the valkey/valkey-go package (https://github.com/valkey/valkey-go).
+package valkey
+
+import (
+	"context"
+	"math"
+	"net"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/valkey-io/valkey-go"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/log"
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/namingschema"
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/telemetry"
+)
+
+const componentName = "valkey-go/valkey"
```

Comment:
```suggestion
const componentName = "valkey-io/valkey-go"
```

Component name and the folder should mirror the module name, stripping the `github.com` prefix.

In this case the folder should be `contrib/valkey-io/valkey-go`